cdc/kv/client.go

```diff
@@ -516,7 +519,7 @@ type eventFeedSession struct {
 	// The channel to send the processed events.
 	eventCh chan<- *model.RegionFeedEvent
 	// The channel to put the region that will be sent requests.
-	regionCh chan singleRegionInfo
+	regionRouter LimitRegionRouter
```

Comment:
I think it's not a channel anymore, could you update the comment?

Reply:
addressed